src/Servers/Kestrel/Core/src/Middleware/LoggingConnectionMiddleware.cs

```diff
@@ -16,7 +16,7 @@ internal class LoggingConnectionMiddleware
 
         public LoggingConnectionMiddleware(ConnectionDelegate next, ILogger logger)
         {
-            _next = next ?? throw new ArgumentNullException(nameof(next));
+            _next = next;
```

Comment:
Was this intentional? If not:

```suggestion
            _next = nextt ?? throw new ArgumentNullException(nameof(next));
```